src/query/query_types.rs

```diff
@@ -5,8 +5,17 @@ pub enum LookupLhs<'a> {
     Parameter(&'a str)
 }
 
+#[allow(dead_code)]
```

Comment:
what dead code are we getting here?

Reply:
Yep, no longer dead code now this PR actually uses it. Removed that